fix(Prometheus): Fixing some errors, urls, typos, etc. (#11566)
newrelic/docs-website: src/content/docs/infrastructure/prometheus-integrations/integrations-list/argocd-integration.mdx

import kubernetesArgoCDDashboard from 'images/kubernetes_screenshot-crop_argocd-dashboard.png'

[Argo CD](https://argoproj.github.io/) is a Kubernetes controller, responsible for continuously monitoring all running applications and comparing their live state to the desired state specified in the Git repository.

> Use New Relic to view a dashboard based on Prometheus metrics. It helps you understand the Argo CD infrastructure of your k8s cluster. With New Relic you can monitor: 

Reviewer: ```suggestion
Use New Relic to view a dashboard based on Prometheus metrics to better understand the Argo CD infrastructure of your k8s cluster. 
```

Author: Thanks!
